runtime/slot-dom-consumer.js

```diff
@@ -180,7 +180,12 @@ export class SlotDomConsumer extends SlotConsumer {
 
   _observe(container) {
     assert(container, 'Cannot observe without a container');
-    this._observer && this._observer.observe(container, {childList: true, subtree: true});
+    this._observer && this._observer.observe(container, {
+      childList: true,
```

Comment:
Have you tried this with `childList` false? It's possible we could reduce observer thrash if we can turn this off, but it would require that attribute mutation occurs on creation (and I don't know if it does).

Reply:
I just ran a quick test and it looks like mutation does NOT fire if you add an element with existing attributes, so I guess we need both.

Reply:
thanks for checking! i'll merge as is for now then, and we can improve later, if we find a way.